### PR TITLE
Adding an NA icon in case of no flag image in Node details

### PR DESCRIPTION
**Pull request by maayarosama**
### Description

Adding an NA icon in case of no flag image in Node details
Screenshot from 2024-08-08 17-08-03

### Related Issues

- #3243
### Documentation PR

For UI changes, Please provide the Documetation PR on [info_grid](https://github.com/threefoldtech/info_grid)

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/playground/src/components/node_details_cards/card_details.vue [hunk 1]
```diff
@@ -40,6 +40,7 @@
 
             <v-tooltip class="overflowText" v-if="item.hint && !item.icon" location="top" :text="item.hint">
               <template #activator="{ props }">
+                <span class="flag-avatar" v-if="item.name === 'Flag' && !item.imgSrc">NA</span>
```
Comment (0oM4R) [inline packages/playground/src/components/node_details_cards/card_details.vue:43]: Would it be better if we added some padding top to the span?

File: packages/playground/src/components/node_details_cards/country_details_card.vue [hunk 2]
```diff
@@ -35,6 +35,8 @@ export default {
     onMounted(mount);
 
     const getNodeDetailsCard = (): NodeDetailsCard[] => {
+      const x = getCountryFlagSrc();
+      console.log("x ", x);
```
Comment (0oM4R) [inline packages/playground/src/components/node_details_cards/country_details_card.vue]: please remove the debug lines

File: packages/playground/src/components/node_details_cards/card_details.vue (hunk 1 above)
Comment (maayarosama) [reply packages/playground/src/components/node_details_cards/card_details.vue:43]: padding is added in flag-avatar class

File: packages/playground/src/components/node_details_cards/country_details_card.vue [hunk 3]
```diff
@@ -48,6 +48,9 @@ export default {
     const getCountryFlagSrc = () => {
       const conuntryCode = getCountryCode(props.node);
 
+      if (conuntryCode.length > 2) {
```
Comment (0oM4R) [inline packages/playground/src/components/node_details_cards/country_details_card.vue:51]: It will be great if you fix this country typo

**Review by 0oM4R — APPROVED** [review]

**Review by Mahmoud-Emad — APPROVED** [review]

**Pull request merged**